### PR TITLE
docs/contributing: Fix broken links from main page

### PR DESCRIPTION
**Pull request by Kaushik-Iyer**
The references to the kind/project, kind/enhancement and kind/bug labels were not working (empty links or 404 errors). Fix the links to these labels mentioned in the "Contributing" page. These links now directly lead to the issues section of the main GitHub repo.

Solves #189 
Signed-off-by: Kaushik Iyer <kushlaser@gmail.com>

### CODE REVIEW

**Review by andraprs — COMMENTED** [review]
Thank you for the updates. :)

The following notes are about the commit message and the PR description:
* In the commit message there is an additional Signed-off-by tag (Signed-off-by: Kaushik-Iyer <84177184+Kaushik-Iyer@users.noreply.github.com>) that should be removed.
* Each line from the body of the commit message should be up to 72 characters [1]. And the commit title should be up to 50 characters; the title is already following that.
* The PR description should reference the corresponding GitHub issue [2], in the "Solves" part.

Other than that, it looks good to me. Thank you for the contribution.

[1] https://cbea.ms/git-commit/
[2] https://github.com/unikraft/docs/issues/189

**Comment by andraprs** [comment]
> How do I remove the additional signed-off-by tag now?

One option is to use the interactive mode of git rebase [1].

[1] https://git-scm.com/docs/git-rebase#_interactive_mode ("If you just want to edit the commit message for a commit, replace the command "pick" with the command "reword".")

**Comment by Kaushik-Iyer** [comment]
I think I changed my commit message but ended up deleting every file. Atleast I got to learn something new. Let me try again, sorry for the inconvenience.

**Comment by andraprs** [comment]
> I think I changed my commit message but ended up deleting every file. Atleast I got to learn something new. Let me try again, sorry for the inconvenience.

Right, I just wanted to mention about the commit contents. The commit message looks fine, thank you. No worries, this is part of further learning and improving.

**Comment by Kaushik-Iyer** [comment]
Till now I have been editing and commiting online, through the github website.

Can you tell me how can I sign-off commits in this way

Or how can I have this codebase locally on my desktop, since cloning the repo only leads to an empty folder

Thank You.

**Comment by andraprs** [comment]
> Till now I have been editing and commiting online, through the github website.
> 
> Can you tell me how can I sign-off commits in this way
> 
> Or how can I have this codebase locally on my desktop, since cloning the repo only leads to an empty folder
> 
> Thank You.

This is one option I've tried out.


```
/* Clone the main Unikraft repository. */

$ git clone https://github.com/unikraft/docs.git

$ cd docs/

$ git status
On branch main
Your branch is up to date with 'origin/main'.

nothing to commit, working tree clean

$ git remote -v
origin	https://github.com/unikraft/docs.git (fetch)
origin	https://github.com/unikraft/docs.git (push)

/* Add your repository fork as another remote. */

$ git remote add Kaushik-Iyer https://github.com/Kaushik-Iyer/docs.git

$ git remote -v
Kaushik-Iyer	https://github.com/Kaushik-Iyer/docs.git (fetch)
Kaushik-Iyer	https://github.com/Kaushik-Iyer/docs.git (push)
origin	https://github.com/unikraft/docs.git (fetch)
origin	https://github.com/unikraft/docs.git (push)

/* Fetch the changes from the forked repository and create a local main branch. */

$ git fetch Kaushik-Iyer

$ git checkout -b Kaushik-Iyer-main Kaushik-Iyer/main

$ git status
On branch Kaushik-Iyer-main
Your branch is up to date with 'Kaushik-Iyer/main'.

nothing to commit, working tree clean

/* Run interactive rebase and choose "edit" for the commit. */

$ git rebase -i origin/main

/* Show the changed files from the top commit. */

$ git show 4d709179bcc4170cbd83ec17a6f56ddf343ad4b1 --pretty="" --name-only

/* Revert the changes from all the files, except the contributing page index. */

$ git show 4d709179bcc4170cbd83ec17a6f56ddf343ad4b1 -- assets/* content/en/community/* layouts/* resources/* scripts/* static/*  | git apply -R

/* It seems the "resources" changes can be reverted with this command as they still remain after the previous command. */

$ git checkout origin/main -- resources/*

/* Commit the changes and then can force push to the forked repository. */

$ git add .

$ git rebase --continue
```
